src/wp-includes/class-wp-script-modules.php

```diff
@@ -151,6 +151,20 @@ public function dequeue( string $id ) {
 		unset( $this->enqueued_before_registered[ $id ] );
 	}
 
+	/**
+	 * Deregister the script module so it will no longer be registered.
```

Comment:
The description from [wp_deregister_script](https://developer.wordpress.org/reference/functions/wp_deregister_script/) could be reused:

```suggestion
	 * Removes a registered script module.
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6061/commits/11d2ea7ecbec9b63f1d20fbda5007397b3153b95